src/test/java/uk/gov/hmcts/reform/professionalapi/controller/advice/ErrorConstantsTest.java

```diff
@@ -1,21 +1,21 @@
-package uk.gov.hmcts.reform.professionalapi.controller;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-import org.junit.Test;
-import uk.gov.hmcts.reform.professionalapi.controller.advice.ErrorConstants;
-
-public class ErrorConstantsTest {
-
-    @Test
-    public void check_constants() {
-
-        assertThat(ErrorConstants.MALFORMED_JSON.getErrorMessage()).isEqualTo("1 : Malformed Input Request");
-
-        assertThat(ErrorConstants.UNSUPPORTED_MEDIA_TYPES.getErrorMessage()).isEqualTo("2 : Unsupported Media Type");
-
-        assertThat(ErrorConstants.INVALID_REQUEST.getErrorMessage()).isEqualTo("3 : There is a problem with your request. Please check and try again");
-
-    }
-
-}
+package uk.gov.hmcts.reform.professionalapi.controller.advice;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+import uk.gov.hmcts.reform.professionalapi.controller.advice.ErrorConstants;
+
+public class ErrorConstantsTest {
+
+    @Test
+    public void check_constants() {
```

Comment:
could you please move to constant package rather than advice

Reply:
This has been addressed here: https://github.com/hmcts/rd-professional-api/pull/450/commits/eee5bbd0d73f59afd9131476d3d95c852735b44b